clients/src/main/java/org/apache/kafka/clients/CommonClientConfigs.java

```diff
@@ -68,7 +68,7 @@
     public static final String METRICS_RECORDING_LEVEL_DOC = "The highest recording level for metrics.";
 
     public static final String METRIC_REPORTER_CLASSES_CONFIG = "metric.reporters";
-    public static final String METRIC_REPORTER_CLASSES_DOC = "A list of classes to use as metrics reporters. Implementing the <code>MetricReporter</code> interface allows plugging in classes that will be notified of new metric creation. The JmxReporter is always included to register JMX statistics.";
+    public static final String METRIC_REPORTER_CLASSES_DOC = "A list of classes to use as metrics reporters. Implementing the <code>MetricsReporter</code> interface allows plugging in classes that will be notified of new metric creation. The JmxReporter is always included to register JMX statistics.";
```

Comment:
We should probably include the full package name `org.apache.kafka.common.metrics.MetricsReporter`.

Reply:
That sounds better. I'm changing the scope of this PR as I noticed that there are quite a few places in the configuration documentation that does not use the full package name.